service/src/test/java/com/ge/predix/acs/privilege/management/dao/GraphResourceRepositoryTest.java

```diff
@@ -398,7 +398,7 @@ public void testPerformance() throws Exception {
 
     @Test(
             expectedExceptions = QueryException.class,
-            expectedExceptionsMessageRegExp = "Graph search failed: traversal limit exceeded.")
+            expectedExceptionsMessageRegExp = "The number of attributes on this resource/subject has exceeded the maximum limit of .*")
```

Comment:
Ideally, the message should identify if its the resource or subject whose attributes exceed the limit. I am ok to defer this change, given the low probability of this error. @ilipkind  ?